### ISSUE TITLE
[Refactor] Delete "Confirm Form" and "Create Bank Form"

### ISSUE THREAD

**Issue by hermannleboss** [globe-and-citizen/cnc-portal#309]
This component is not realy needed. 
We just use it for the modal. 

And we already have a modal. 

We can simply use the modal mechanism to have the same result. 

Look in member card and Team Meta Section

![Image](https://github.com/user-attachments/assets/18e01861-047b-44de-97fb-0b8a3cd90f47)

![Image](https://github.com/user-attachments/assets/ef9eaca2-f579-4706-8f94-7130c448e1cd)

![Image](https://github.com/user-attachments/assets/a6da4845-f617-410d-ab04-68a50994fd4f)

**Comment by dasarathg68** [comment]
You mean you want to directly wrap the contents of DeleteConfirmForm in ModalComponent instead of a separate component for it?
Something like:
```typescript
<ModalComponent>
// contents defined inline
</ModalComponent>

**Comment by hermannleboss** [comment]
Yes it more simple like that. Because the content are not reused anywhere

**Comment by dasarathg68** [comment]
> Yes it more simple like that. Because the content are not reused anywhere

But DeleteConfirmForm is used in multiple places. Eg: Delete team, member
